### PR TITLE
Split AE and Reviewer FAQ

### PR DESCRIPTION
**Pull request by neubig**
This splits off the AE and reviewer FAQ and does some other miscellaneous organization, content fixes.

### CODE REVIEW

File: authors.md [hunk 1]
```diff
@@ -37,17 +33,17 @@ ACL Rolling Review offers the following:
 **Q**: I have further questions about ARR submission, where do I contact?<br>
 **A**: Please contact: `support@aclrollingreview.org`
 
-# Anonymity Requirement
+## Anonymity Requirement
 
 **Q**: When am I allowed to post an anonymous preprint?
-**A**: Please see the [Call for Papers](https://aclrollingreview.org/cfp) for a detailed policy?
+**A**: Please see the [Call for Papers](/cfp) for a detailed policy?
```
Comment (nschneid) [inline authors.md:39]: ? should be .

File: aes.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+# FAQ
+
+This FAQ is intended to help reviewers and action editors navigate OpenReview and the review process. Please direct inquiries to: `support@aclrollingreview.org`
+
+## Timeline
+
+### What is the timeline?
+
+See the [dates and venues](/dates) page.
```
Comment (nschneid) [inline aes.md:40]: It would make it easier to also have specific dates (rather than offsets) in the emailed invitation to AEs.

File: aes.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+## Communication
+
+### How can I communicate with reviewers?
+You may communicate with reviewers using the discussion forum or by sending them email directly (reviewers and AEs are not blind to each-other).
+
+To send a note via the discussion forum In OpenReview, you can use a button labeled "Official Comment" at the top of the paper forum.
```
Comment (nschneid) [inline aes.md:47]: In -> in

File: aes.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+## Timeline
+
+### What is the timeline?
+
+See the [dates and venues](/dates) page.
+
+## Communication
+
+### How can I communicate with reviewers?
+You may communicate with reviewers using the discussion forum or by sending them email directly (reviewers and AEs are not blind to each-other).
```
[the remark this answers is not in the document]
Comment (nschneid) [reply aes.md:45]: Are there things AEs should be on the lookout for that would be inappropriate in reviews, and ask the reviewers to edit? E.g. vague criticisms (ask the reviewer to be more concrete), overly harsh tone....

File: aes.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+### In what order are alternative reviewers for a submission displayed (eg alphabetical, by match score, random)?
+You can choose several orders. If you choose "affinity score" they will be sorted by the matching score given by an ML model (specifically, as of September this is the [SPECTRE-MFR model](https://github.com/openreview/openreview-expertise) provided by OpenReview).
+
+### Can we see the number of submissions a reviewer is already assigned?
+Yes, look for the number next to "assignments" by their name. Please do consider this when making new assignments.
+
+### Can we see author information?
+No, we are managing a two-way anonymized process up through action editors. It is not possible for you to assign a reviewer who has a COI that is automatically identifiable.
+
```
Comment (nschneid) [inline aes.md:113]: What about resubmissions? Is it important to assign the original reviewers if possible (or is it OK to replace, for example, a positive but less informative reviewer)? Note that the platform provides access to the new paper as well as authors' summary of changes. Any other special considerations for handling resubmissions?

File: aes.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+You are free to decline to review papers, but keep in mind that ARR will be the sole reviewing process for ACL/NAACL in 2022, so a) you won't get separate review requests from conference organizers, and b) the volume should come down in the following months.
+
+### Why did I get an invitation to review something when the deadline is already past?
+It is possible that an AE may send as an emergency review request for one of the prior month submissions that we're running late with. There is not a strict deadline in this case, but the sooner you could provide the review, the better.
+
+### I got a review invitation that I’d like to decline, but when I click the decline button, I get the following error message on Openreview: "Wrong key, please refer back to the recruitment email." What do I have to do to be able to accept/decline invitations? Is this a known problem?"	
+This is a known issue as the link (as opposed to the link *text*) uses %-encoding for umlauts (or even any non-ASCII characters). Openreview currently send emails in plain text and they are planning to send them in HTML format with all the links already translated, We hope that will solve the problem.
```
Comment (nschneid) [inline aes.md:144]: typo: we

File: authors.md (hunk 1 above)
Comment (neubig) [reply authors.md:39]: Thanks, fixed!

File: aes.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+Yes, see [here](/people).
+
+### I got 4 papers. Should I expect this load in the future?
+We aim for the average monthly load in any given month to be 1-2 papers, but in certain periods (close to large conferences), it's possible that the load may be higher, and in slow periods the load may be lower.
+
+### Am I expected to accept all invitations? or should I select a subset that I am most interested in?	
+You are free to decline to review papers, but keep in mind that ARR will be the sole reviewing process for ACL/NAACL in 2022, so a) you won't get separate review requests from conference organizers, and b) the volume should come down in the following months.
+
+### Why did I get an invitation to review something when the deadline is already past?
+It is possible that an AE may send as an emergency review request for one of the prior month submissions that we're running late with. There is not a strict deadline in this case, but the sooner you could provide the review, the better.
```
[the remark this answers is not in the document]
Comment (neubig) [reply aes.md:141]: Yes, I rewrote this.

File: aes.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (neubig) [reply aes.md:45]: These are good questions, but I'll put this as an issue to fix in the future.

File: aes.md (hunk 5 above)
Comment (neubig) [reply aes.md:113]: Similarly started a separate issue.

File: reviewers.md [hunk 8]
```diff
@@ -4,52 +4,81 @@ title: "For Reviewers and Action Editors"
...
 
-Each month, starting around 24 days after the submission deadline, the action editor will remind any reviewer who has not yet completed their review.
+Each month, around 30 days after the submission deadline, the action editor will read the reviews and discuss with the reviewers if necessary. Reviewers should be responsive during this time.
 
-Each month, between 30 and 35 days after the submission deadline, the action editor will read the reviews, discuss with the reviewers if necessary, and:
-* Summarize the review results in a meta-review.
-* Perform a compliance check, checking the paper meets formatting and other requirements.
+So, reviewers will want to be especially available from the 15th to the 22nd of each month.
+ 
+# Reviewer Time Committment
```
[the remark this answers is not in the document]
Comment (neubig) [reply reviewers.md:18]: Fixed.

**Pull request merged**